.github/workflows/build.yaml

```diff
@@ -0,0 +1,31 @@
+name: Discord Bot CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [develop]
```

Comment:
Can u explain me what is the workflow for the repo?
Owned branch -- PR --> develop -- ? --> main
Maybe also include it on the README for reference if I (or someone else... ex di) need to help you in the future.

Reply:
I will explain all the setup in the README in a future PR, right now I'm just trying to see how things are going to work.